source/conf.py

```diff
@@ -135,7 +135,7 @@ def build_partials(app, env, docnames):
       partial = open('{}{}{}'.format(source_path, docname, '.rst'), 'r').read()
```

Comment:
I wouldn't worry about them. I'm sure there is a way to do this in Python but it isn't occurring to me off the top of my head.